types/types.go

```diff
@@ -20,6 +20,7 @@ type HelmRepo struct {
 
 type Release struct {
 	Name       string     `yaml:"name"`
+	Deployment string     `yaml:"deployment"`
```

Comment:
Can we rename this to `deploymentMode` or `deploymentMethod`? I don't think the current name captures the functionality.

Reply:
Fixed: 0395f1b01df2ee0ec09afe75356be7141dea7a38